Source/Dafny/Util.cs

```diff
@@ -858,7 +858,7 @@ public class ExpressionTester {
           }
           // function is okay, so check all NON-ghost arguments
           isCompilable = CheckIsCompilable(callExpr.Receiver, codeContext);
-          for (var i = 0; i < callExpr.Function.Formals.Count; i++) {
+          for (var i = 0; i < callExpr.Function.Formals.Count && i < callExpr.Args.Count; i++) {
             if (!callExpr.Function.Formals[i].IsGhost) {
```

Comment:
```suggestion
          for (var i = 0; i < callExpr.Function.Formals.Count; i++) {
            if (!callExpr.Function.Formals[i].IsGhost  && i < callExpr.Args.Count) {
```

Reply:
There is no way your suggestion does not imply mine, but for maintenance it's surely better.